samples: broadcast_config_tool: Add option for broadcast_ID (#17440)
nrfconnect/sdk-nrf: samples/bluetooth/broadcast_config_tool/README.rst

This command sets the presentation delay for the subgroup 0 in BIG0 to 40 ms (40000 us).

fixed_id
========

Set the broadcast ID for a BIG.

> The broadcast ID is a 3 octets long value that is used to identify the broadcast.

Reviewer: ```suggestion
The broadcast ID is used to identify the broadcast. Its value is long three octets.
```

Author: Should that be: `three octets long`?

The default setting is to use a random broadcast ID that will change each time the broadcaster is started.

Usage:

.. code-block:: console

   bct fixed_id <0/1> <BIG index> [broadcast_id in hex (3 octets)]

Examples:

.. code-block:: console

   bct fixed_id 1 0 0xAA1234

> This command sets a fixed broadcast ID for the BIG 0 to 0xAA1234.

Reviewer: ```suggestion
This command sets a fixed broadcast ID for the BIG 0 to ``0xAA1234``.
```

This command sets the presentation delay for the subgroup 0 in BIG0 to 40 ms (40000 us).

fixed_id
========

Set the broadcast ID for a BIG.
The broadcast ID is used to identify the broadcast.

> Its value is 3 octets long.

Reviewer: ```suggestion
Its value is three octets long.
```
